orca-web/lib/systemd/system/orca.service

```diff
@@ -0,0 +1,14 @@
+[Unit]
+Description=Spinnaker Orca
+PartOf=spinnaker.service
+
+[Service]
+ExecStart=/opt/orca/bin/orca 2>&1 /var/log/spinnaker/orca/orca.log
+WorkingDirectory=/opt/orca/bin
+SuccessExitStatus=143
```

Comment:
heh

Reply:
Nothing suspicious here :)

Just so when someone in the future comes looking:

https://docs.spring.io/spring-boot/docs/current/reference/html/deployment-install.html#deployment-systemd-service